Web/starter_project_vanguard/components/common/Navbar.tsx

```diff
@@ -0,0 +1,50 @@
+import clsx from 'clsx'
+import { Montserrat } from 'next/font/google'
+import Image from 'next/image'
+import Link from 'next/link'
+
+import { navItems } from '@/utils/NavItems'
+import { useRouter } from 'next/router'
+import logo from '../../public/logo.png'
+import { Button } from '../example/Button'
+
+const montserrat = Montserrat({ subsets: ['latin'] })
+
+const Navbar = () => {
+  const router = useRouter()
+
+  const currSegment = router.pathname.split('/')[1] || ''
+
+  console.log(currSegment)
+
+  return (
+    <div className={montserrat.className}>
+      <div className="flex justify-between">
+        <Link href="/">
+          <Image src={logo} className="" alt="A2SV Logo" />
```

Comment:
[nit] is there need for a className that has no value in it?

Reply:
the navbar item name's font is Montserrat, which is why I used it here.

Reply:
I am not talking about the font, I mean why do we have a className on `<Image src={logo} className="" alt="A2SV Logo" />`